calculator/calc_v2.py

```diff
@@ -0,0 +1,63 @@
+import custom_exceptions
+
+CALCULATION_OPERATIONS = ('+', '-', '*', '/', '**')
+
+
+def get_parameters():
+    while True:
+        num_one = input('Enter first num: ')
```

Comment:
hmm
num_one, num_two, operation = input('EFN'),  input('ESN'), input('EO')
if 'exit' in num_one, num_two, operation:
    quit()